contributors/contributors.yaml

```diff
@@ -1,120 +1,126 @@
 - id: tommyvedvik
   name: Tommy Vedvik
-  bio: "Co-creator of @gridsome. UX & Design."
```

Comment:
This is introducing a lot of unrelated churn. Can we stick with the double quotes throughout?

Reply:
oh wow, thanks for pointing that out. That was my prettier config.